tests/chainer_tests/functions_tests/normalization_tests/test_layer_normalization.py

```diff
@@ -0,0 +1,110 @@
+import unittest
+
+import numpy
+
+import chainer
+from chainer import cuda
+from chainer import functions
+from chainer import gradient_check
+from chainer import testing
+from chainer.testing import attr
+from chainer.testing import condition
+
+
+def _batch_normalization(expander, gamma, beta, x, mean, var):
+    mean = mean[expander]
+    std = numpy.sqrt(var)[expander]
+    y_expect = (gamma[expander] * (x - mean) / std + beta[expander])
+    return y_expect
+
+
+@testing.parameterize(*(testing.product({
+    'batchsize': [1, 5],
+    'size': [10, 20],
+    'dtype': [numpy.float32],
+})))
+class TestBatchNormalization(unittest.TestCase):
+
+    def setUp(self):
+        shape = (self.batchsize, self.size)
+        size = (numpy.prod(shape) // shape[0],)
+        x = numpy.random.uniform(-1, 1, shape).astype(self.dtype)
+        gamma = numpy.random.uniform(-1, 1, size).astype(self.dtype)
+        beta = numpy.random.uniform(-1, 1, size).astype(self.dtype)
+        self.args = [x, gamma, beta]
```

Comment:
Use a tuple if you do not modify the content of an iterable object.

Reply:
Fixed.